pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/KeyValueSchema.java

```diff
@@ -236,4 +245,36 @@ public String getTopicName() {
             }
         });
     }
+
+
+    public static String getKeyValueSchemaString(SchemaData schemaData) {
```

Comment:
@congbobo184 

Use `KeyValueSchemaInfo.decodeKeyValueSchemaInfo`:

- covert SchemaData to SchemaInfo
- KeyValue<SchemaInfo, SchemaInfo> kvsSchemaInfo = KeyValueSchemaInfo.decodeKeyValueSchemaInfo(schemaInfo);
- Use ObjectMapper to convert kvSchemaInfo into a json string.

The schemaInfo serializer and deserializer is already defined in SchemaUtils. You can modify it to handle KEY_VALUE correctly.